packages/components/src/unit-control/styles/unit-control-styles.ts

```diff
@@ -96,6 +119,7 @@ export const UnitSelect = styled.select< SelectProps >`
 		cursor: pointer;
 		border: 1px solid transparent;
 		height: 100%;
+		font-family: inherit;
```

Comment:
Similar to #38969, this was incorrectly rendering in Arial (per Chrome user agent style).